docs/dev/table/connect.md

```diff
@@ -276,6 +276,40 @@ tables:
         type: VARCHAR
 {% endhighlight %}
 </div>
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- CREATE a 010 version Kafka table start from the earliest offset(as table source) and append mode(as table sink).
+CREATE TABLE MyUserTable (
+  `user` bigint,
+  message varchar,
+  ts varchar
+) WITH (
+  -- declare the external system to connect to
+  'connector.type' = 'kafka',
+  'connector.version' = '0.10',
+  'connector.topic' = 'topic_name',
+  'connector.startup-mode' = 'earliest-offset',
+  'connector.properties.0.key' = 'zookeeper.connect',
+  'connector.properties.0.value' = 'localhost:2181',
+  'connector.properties.0.key' = 'bootstrap.servers',
+  'connector.properties.0.value' = 'localhost:9092',
+  'update-mode' = 'append',
+  -- declare a format for this system
+  'format.type' = 'avro',
+  'format.avro-schema' = '{
+                            \"namespace\": \"org.myorganization\",
```

Comment:
why do we need a backslash for double quotes here, if we use `'` as surrounding quotes?

Reply:
This is a quote for JSON string, not a sql identifier quote.